tests/qos/test_qos_sai.py

```diff
@@ -1031,8 +1034,9 @@ def testQosSaiLossyQueueVoq(
             Raises:
                 RunAnsibleModuleFail if ptf test fails
         """
-        if dutTestParams["basicParams"]["sonic_asic_type"] != "cisco-8000":
-            pytest.skip("Lossy Queue Voq test is not supported")
+        if not get_src_dst_asic_and_duts['single_asic_test']:
+             pytest.skip("LossyQueueVoq: This test is skipped for now, will"
+                 " be re-enabled for RH cards.")
```

Comment:
VOQ1: this one will change to validate fair voq so we should update skip reason accordingly.